tests/plugins/sheetview.test.ts

```diff
@@ -136,24 +136,24 @@ describe("Viewport of Simple sheet", () => {
     expect(model.getters.getActiveMainViewport()).toMatchObject({
       left: 0,
       right: 10,
-      top: 57,
+      top: 59,
```

Comment:
Together with other top and bottom changes with the differences of only 2 or 3 pixels, I don't understand why these differences happen.

Reply:
Can this be explained by the changes you made in the internal_viewport.ts file ?

Reply:
@anhe-odoo wan to help out on this? :D